vendor/modules.txt

```diff
@@ -3,7 +3,7 @@ github.com/Azure/go-ansiterm/winterm
 github.com/Azure/go-ansiterm
 # github.com/BurntSushi/toml v0.3.1
 github.com/BurntSushi/toml
-# github.com/Microsoft/go-winio v0.4.11
+# github.com/Microsoft/go-winio v0.4.12
```

Comment:
This commit doesn't have any code updates. Looks like the dep is commented out?

Reply:
The code which has changed within `go-winio` from v0.4.11 to v0.4.12 is not used by CRI-O and therefore not vendored or updated. The comment just indicates the section within the `modules.txt` file, whereas the lines below summarize the import paths:

```
github.com/Microsoft/go-winio
github.com/Microsoft/go-winio/archive/tar
github.com/Microsoft/go-winio/backuptar
```

So this should be fine :)